lib/format.js

```diff
@@ -0,0 +1,19 @@
+const util = typeof window === 'undefined' ? require('util') : null;
```

Comment:
I was thinking about whether we should have a tiny package with some environment info on it, taking the place of `Meteor.isClient`, etc. to improve code readability.

```js
import {isBrowser, isNode} from 'isoinfo'
```

I didn't find anything exactly the same on npm, but there might be something. Found this fun little one that is almost there: https://github.com/enki/isnode/blob/master/isnode.js